src/styles/GeneralOwnerDashboard.css

```diff
@@ -10,15 +10,13 @@
   background: white;
 }
 
-.dashboard .dash-solar-details-cont,
-.dash-announcements-cont {
+.dashboard {
   margin-top: 55px;
```

Comment:
Replace all margin tags from this class and replace with `margin: 60px`

Reply:
Also remove `width` from `.dashboard`. This should make the white box a bit nicer.